pdsfile/pdsfile.py

```diff
@@ -620,6 +620,72 @@ def new_pdsfile(self, key=None, copypath=False):
 
         return this
 
+    ######################################################################################
+    # Set parameters for both Pds3File and Pds4File
+    ######################################################################################
+    @classmethod
+    def use_shelves_only(cls, pds3_status=True, pds4_status=True):
```

Comment:
For this and `require_shelves` the point is to set both subclasses to the same value. If there are two parameters, we might as well just call the two subclass functions directly.